test/parallel/test-whatwg-encoding-custom-textdecoder.js

```diff
@@ -201,8 +201,13 @@ if (common.hasIntl) {
 }
 
 if (common.hasIntl) {
-  const decoder = new TextDecoder('Shift_JIS');
-  const chunk = new Uint8Array([-1]);
-  const str = decoder.decode(chunk);
-  assert.strictEqual(str, '\ufffd');
+  try {
+    const decoder = new TextDecoder('Shift_JIS');
+    const chunk = new Uint8Array([-1]);
+    const str = decoder.decode(chunk);
+    assert.strictEqual(str, '\ufffd');
+  } catch (e) {
+    // Encoding may not be available, e.g. small-icu builds
+    assert(e.code, 'ERR_ENCODING_NOT_SUPPORTED');
```

Comment:
I assume this is not supposed to test the truthiness of `e.code`, so probably:

```suggestion
    assert.strictEqual(e.code, 'ERR_ENCODING_NOT_SUPPORTED');
```